.env.example

```diff
@@ -0,0 +1,3 @@
+DB_HOST=
+DB_USERNAME=
+DB_PASSWORD=
```

Comment:
I don't think we need this example file here, please add this as part of the instructions in the README.md

Reply:
For example, add instructions in the readme, something like this:
```md
Add a file with the credentials you need under `.env/secrets`:
DB_HOST=<host>
DB_USERNAME=<username>
DB_PASSWORD=<password>
```

Reply:
done